source/lib/core/perfetto.cpp

```diff
@@ -263,6 +263,30 @@ post_process(tim::manager* _timemory_manager, bool& _perfetto_output_error)
                 _timemory_manager->add_file_output("protobuf", "perfetto", _filename);
         }
         ofs.close();
+
+        if(dmp::rank() == 0)
+        {
+            const char* file_path   = _filename.c_str();
+            char*       folder_path = strdup(file_path);
+            char*       last_slash  = strrchr(folder_path, '/');
+            if(last_slash != nullptr)
+            {
+                *last_slash = '\0';  // Terminate the string at the last slash
+            }
+
+            // Execute the merge script
+            std::string command =
+                "merge_multiprocess_output.sh '" + std::string(folder_path) + "'";
```

Comment:
```suggestion
                "merge-multiprocess-output.sh '" + std::string(folder_path) + "'";
```
Updated to account for name change.